services/github/github-last-commit.service.js

```diff
@@ -106,7 +106,7 @@ export default class GithubLastCommit extends GithubAuthV3Service {
 
   async handle({ user, repo, branch }, queryParams) {
     const { path, display_timestamp: displayTimestamp } = queryParams
-    const body = await this.fetch({ user, repo, branch, path })
+    const body = await this.fetch({ user, repo, branch, path, per_page: 1 })
```

Comment:
Does this not need to go in `fetch` [here](https://github.com/chris48s/shields/blob/87b9049eb14a7ac71fd58cea8f652e2d0a296015/services/github/github-last-commit.service.js#L101) like your GitLab change?

Reply:
good catch! thanks for the review